packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -112,7 +112,9 @@ export class FetchInstrumentation extends InstrumentationBase<
   ): void {
     const parsedUrl = web.parseUrl(response.url);
     span.setAttribute(HttpAttribute.HTTP_STATUS_CODE, response.status);
-    span.setAttribute(HttpAttribute.HTTP_STATUS_TEXT, response.statusText);
+    if (response.statusText != undefined) {
```

Comment:
nit: !== undefined

Reply:
I think we may end up stuck in a loop https://github.com/open-telemetry/opentelemetry-js/pull/1851#discussion_r561226547